### ISSUE TITLE
Feature: Query View - Add Copy All button

### ISSUE THREAD

**Issue by darrylthom** [luusluus/vscode-parquet-visualizer#82]
The copy button currently only copies what is visible. It would be useful to have a "Copy All" button to copy the entire dataset to your clipboard. Perhaps it would also make more sense at that point to rename Copy to "Copy Visible".

**Comment by luusluus** [comment]
Makes sense. By the way, thank you for the thorough testing and reporting of bugs. I appreciate it a lot!
